pages/linux/numactl.md

```diff
@@ -0,0 +1,16 @@
+# numactl
+
+> Control NUMA policy for processes or shared memory.
+> More information: <https://man7.org/linux/man-pages/man8/numactl.8.html>.
+
+- Run command on node 0 with memory allocated on node 0 and 1:
+
+`numactl --cpunodebind={{0}} --membind={{0,1}} -- {{command}} {{command_arguments}}`
+
+- Run command on CPUs (cores) 0-4 and 8-12 of the current cpu-set:
+
+`numactl --physcpubind={{+0-4,8-12}} {{command}} {{command_arguments}}`
```

Comment:
```suggestion
`numactl --physcpubind={{+0-4,8-12}} -- {{command}} {{command_arguments}}`
```

Reply:
Have you ever had issues without the --? If so, where?

Reply:
I haven't ever even used this command :D But I think it's always best to have the `--` because a great many commands run subcommands like this and you can use `--` with them.  It's also more verbose, which I think would be useful to new users of the command (like me for example :)

Reply:
IMO it also makes it quite a bit clearer what args are for `numactl` and what are for the command you're running with numactl.

Reply:
I understand! Thank you for taking some time to explain your thoughts on the subject; greatly appreciated :D